sql/core/src/test/scala/org/apache/spark/sql/DatasetBenchmark.scala

```diff
@@ -135,12 +175,26 @@ object DatasetBenchmark {
     benchmark.run()
 
     /*
+    Java HotSpot(TM) 64-Bit Server VM 1.8.0_60-b27 on Mac OS X 10.11.4
+    Intel(R) Core(TM) i7-4960HQ CPU @ 2.60GHz
     back-to-back filter:                Best/Avg Time(ms)    Rate(M/s)   Per Row(ns)   Relative
     -------------------------------------------------------------------------------------------
     Dataset                                   585 /  628         17.1          58.5       1.0X
     DataFrame                                  62 /   80        160.7           6.2       9.4X
     RDD                                       205 /  220         48.7          20.5       2.8X
     */
     benchmark2.run()
+
+    /*
+    Java HotSpot(TM) 64-Bit Server VM 1.8.0_60-b27 on Mac OS X 10.11.4
+    Intel(R) Core(TM) i7-4960HQ CPU @ 2.60GHz
+    aggregate:                          Best/Avg Time(ms)    Rate(M/s)   Per Row(ns)   Relative
+    -------------------------------------------------------------------------------------------
+    DataFrame sum                             137 /  314         72.7          13.7       1.0X
```

Comment:
we need to increase the datasize by at least an order of magnitude (maybe even two orders of magnitude).

the dataframe sum should be between 1ns - 2ns per row.